docs/guides/voyage/voyage.md

```diff
@@ -70,6 +70,23 @@ Voyages have specific requirements, milestones, and ***require a commitment of 8
   | Data Scientist | Data Scientists help the team to add value to the project by using advanced data analysis and/or ML to create dashboards and data visualizations based on data captured by the application. |
 </details>
 
+<details>
+  <summary>Tier Definitions 🪜</summary>
+  <br/>
+
+  At Chingu we use three _tiers_ to group people into teams based on their
+  current general level of skill and expertise. Our tiers are:
+
+  * **Tier 1** - Getting started in your role
+  * **Tier 2** - Intermediate in your role
+  * **Tier 3** - Experiences in your role
```

Comment:
"Experience in your role" - Feels like something is missing. Perhaps something like "Advanced skill-level in your role"

Reply:
Thanks. I've updated this to be 'Experienced skill level in your role"

Reply:
Looks good!